.renovaterc.json5

```diff
@@ -0,0 +1,42 @@
+{
+  assignees: [
+    "kevinmatthes",
+  ],
+  automerge: false,
+  commitMessageAction: "update",
+  commitMessageLowerCase: "never",
+  commitMessagePrefix: "⬆️",
+  dependencyDashboard: true,
+  dependencyDashboardAutoclose: true,
+  dependencyDashboardLabels: [
+    "dependencies",
+  ],
+  labels: [
+    "dependencies",
+  ],
+  lockFileMaintenance: {
+    commitMessageAction: "maintain Cargo.lock",
+    commitMessagePrefix: "🤖",
+    enabled: true,
+    schedule: "0 0 * * MON",
+  },
+  packageRules: [
+    {
+      addLabels: [
+        "rust",
+      ],
+      matchManagers: [
+        "cargo",
+      ],
+    },
+    {
+      addLabels: [
+        "github_actions",
+      ],
+      matchManagers: [
+        "github-actions",
+      ],
+    },
+  ],
```

Comment:
Search for updates to all configured Rust dependencies and GitHub Actions and give the respective labels in addition to "dependencies" to the resulting PRs.